demo/admin/src/products/categories/ProductCategoriesPage.tsx

```diff
@@ -45,7 +49,27 @@ const ProductCategoriesPage: React.FC = () => {
                     {(selectedId) => (
                         <SaveBoundary>
                             <FormToolbar />
-                            <ProductCategoryForm id={selectedId} />
+                            <RouterTabs>
+                                <RouterTab
+                                    forceRender={true}
+                                    path=""
+                                    label={intl.formatMessage({ id: "products.editProductCategory.formTab", defaultMessage: "Product category" })}
+                                >
+                                    <ProductCategoryForm id={selectedId} />
+                                </RouterTab>
+                                <RouterTab
+                                    forceRender={true}
+                                    path="/assigned-products"
+                                    label={intl.formatMessage({
+                                        id: "products.editProductCategory.assignedProducts",
+                                        defaultMessage: "Assigned Products",
+                                    })}
+                                >
+                                    <Box sx={{ height: "100vh" }}>
```

Comment:
This shouldn't be necessary

Reply:
but it is, because `AssignedProductsTab` does only contain the grid and this does not have any height.

Reply:
Using `MainContent` would be correct in this case. 

```suggestion
                                    <MainContent fullHeight disablePadding>
```

_I'm currently looking into ways to make all this styling unnecessary, as this is really confusing when you just want to add a Grid somewhere._